announcements/2019-06-06.rst

```diff
@@ -0,0 +1,23 @@
+Update of Unit-e Alpha Testnet
+==============================
+
+.. feed-entry::
+   :date: 2019-06-06 15:00
+
+On June 12 2019 there will be a reset of the alpha testnet as part of a
+scheduled maintenance update after running it for more than half a million
+blocks. The update will bring the nodes run by the Unit-e development team to
```

Comment:
Typo (I think) `run` -> `ran`

Reply:
It's present, not past, isn't it. Or should it maybe be "the nodes being run"?

Reply:
After thinking about it for a while I think you are correct - nodes run by. It's better without the being as it would create a passive form.